src/quo2/components/wallet/network_bridge/view.cljs

```diff
@@ -0,0 +1,45 @@
+(ns quo2.components.wallet.network-bridge.view
+  (:require
+    [clojure.string :as string]
+    [quo2.components.icon :as icon]
+    [quo2.components.markdown.text :as text]
+    [quo2.foundations.colors :as colors]
+    [quo2.foundations.resources :as resources]
+    [quo2.theme :as quo.theme]
+    [react-native.core :as rn]
+    [quo2.components.wallet.network-bridge.style :as style]))
+
+(defn network-bridge-internal
+  [{:keys [theme network state amount]}]
+  (let [network-text (if (= network :ethereum) "Mainnet" (string/capitalize (name network)))]
```

Comment:
Is "Mainnet" the same across all languages? should it be translated?

Reply:
I don't think Mainnet is an English word

Reply:
let's confirm this with design/copy 👍

Reply:
John confirmed it stays the same across different languages

Reply:
Thanks for checking :)